uninstall.md

```diff
@@ -17,16 +17,24 @@ yum erase "*openshift*" "docker*"
 systemctl reset-failed
 systemctl daemon-reload
 
-# Remove data and configuration
+# Remove data, config, and training materials
 rm -rf /var/lib/openshift/
 rm /etc/sysconfig/*openshift*
 rm /etc/sysconfig/docker*
 rm -rf /etc/openshift/*
 rm -rf /root/.config/openshift
+rm -rf /root/training
+rm -rf /root/openshift-ansible
+rm -rf /etc/ansible
```

Comment:
It might be safer just to tell people to yum erase ansible on their ansible host and then rm -rf /etc/ansible.